code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -263,8 +263,9 @@
 		hijack(xeno)
 		return
 
+#define HIJACK_EVILUTION_BUFF 10
+#define HIJACK_EVILUTION_TIME 3 MINUTES
```

Comment:
Move these to https://github.com/cmss13-devs/cmss13/blob/a730d2b5cbf69388d919396e7a94b03cd5dcb8e9/code/__DEFINES/xeno.dm#L171-L178 with autodoc comments to be with the rest of most tweakable settings for xenos. Or a different define file if you find one more appropriate.